plugin/trino-mongodb/src/main/java/io/trino/plugin/mongodb/MongoMetadata.java

```diff
@@ -67,6 +68,8 @@
 {
     private static final Logger log = Logger.get(MongoMetadata.class);
 
+    private static final int MAX_QUALIFIED_IDENTIFIER_LENGTH = 120;
```

Comment:
nit: ` Check target table name when renaming a table in MongoDB ` -> ` Check target table name length when renaming a table in MongoDB `

Reply:
The MongoDB docs talk about bytes and it seems they don't disallow multibyte characters so instead of checking string length we should check bytes probably.

Reply:
Good catch. Let me fix.

Reply:
nit: the var should probably be renamed since it's bytes now instead of length.